embedded-keycloak/src/test/java/com/playtika/test/keycloak/KeycloakTestApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.playtika.test.keycloak;
+
+import org.keycloak.adapters.springsecurity.KeycloakConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+
+@Import(KeycloakConfiguration.class)
+@KeycloakConfiguration
+@SpringBootApplication
+public class KeycloakTestApplication {
+    // nothing here so far
```

Comment:
no need to have this comment

Reply:
Really…

BTW: I mean Google code conventions say having explicit comments implicitly makes clear nothing  has been forgotten in empty implementation.